site/content/en/docs/tutorials/setup_minikube_in_china.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "minikube start"
+linkTitle: "Get Started!"
+weight: 1
+---
+
+This Tutorial explains how to setup minikube in China.
+
+<h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">1</strong></span>Installation</h2>
+
+### Binary download
+
+For more detail please refer to [minikube installation guide]({{<ref "/docs/start">}})
+
+
+<h2 class="step"><span class="fa-stack fa-1x"><i class="fa fa-circle fa-stack-2x"></i><strong class="fa-stack-1x text-primary">2</strong></span>Start your cluster</h2>
```

Comment:
let not use HTML in markdown for consistancy.

Reply:
sure

Reply:
sure

Reply:
removed the HTML.